File: Frontend/src/components/RegularHours.js
import React from "react";

const RegularHours = ({ hours }) => {
  return (
    <div>
      <h2>Regular Hours</h2>
      <p>Weekday: {hours.regularWeekdayHours}</p>
      <p>Friday: {hours.friday}</p>
      <p>Weekend: {hours.weekendHours}</p>
    </div>
  );
};

export default RegularHours;

File: Backend/src/main/java/com/example/demo/Entity/RegularHours.java
package com.example.demo.Entity;


import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.Data;

@Entity
@Data
public class RegularHours {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;

    private String regularWeekdayHours;
    private String friday;
    private String weekendHours;

}
